templating.rst

```diff
@@ -6,74 +6,75 @@ Creating and Using Templates
 
 As you know, the :doc:`controller </controller>` is responsible for
 handling each request that comes into a Symfony application. In reality,
-the controller delegates most of the heavy work to other places so that
-code can be tested and reused. When a controller needs to generate HTML,
-CSS or any other content, it hands the work off to the templating engine.
+the controller delegates most of the heavy work to other places. When
+a controller needs to generate HTML, it hands the work off to the
+*templating* engine.
+
 In this chapter, you'll learn how to write powerful templates that can be
 used to return content to the user, populate email bodies, and more. You'll
 learn shortcuts, clever ways to extend templates and how to reuse template
 code.
 
+Rendering a Template
+--------------------
+
+Typically, you'll render a template from inside of your controller::
+
+    // src/AppBundle/Controller/LuckyController.php
+    public function numberAction()
+    {
+        $number = mt_rand(0, 10);
+
+        // renders app/Resources/views/lucky/number.html.twig
+        return $this->render('lucky/number.html.twig', array(
+            // pass two variables into your template
+            'randomNumber' => $number,
+            'people' => array('Javier', 'Christian, 'Wouter')
```

Comment:
Just a proposal ... what about removing our names here and using anonymous names with some more diversity (women, some non-European + non-American names, etc.)  For example:

```php
'people' => array('Anna', 'Cheng', 'John')
```